From: Brian Norris <computersforpeace@gmail.com>
Date: Wed, 16 Aug 2023 23:09:30 -0700
Subject: [PATCH 2/3] flashrom: Update args to upstream

* --fmap: needed; it's not implicit
* -r /dev/null: we need to have *some* target file for the read; it's
  not an optional arg

Signed-off-by: Brian Norris <computersforpeace@gmail.com>
---
 host/lib/flashrom.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/host/lib/flashrom.c
+++ b/host/lib/flashrom.c
@@ -122,8 +122,10 @@ vb2_error_t flashrom_read(struct firmwar
 		FLASHROM_EXEC_NAME,
 		"-p",
 		image->programmer,
+		"--fmap",
 		"-r",
-		region ? "-i" : tmpfile,
+		region ? "/dev/null" : tmpfile,
+		region ? "-i" : NULL,
 		region ? region_param : NULL,
 		NULL,
 	};
